waku/waku_rln_relay/protocol_types.nim

```diff
@@ -49,8 +53,21 @@ type RateLimitProof* = object
   ## nullifier enables linking two messages published during the same epoch
   ## see details in https://hackmd.io/tMTLMYmTR5eynw2lwK9n1w?view#Nullifiers
   nullifier*: Nullifier
-  ## Application specific RLN Identifier
-  rlnIdentifier*: RlnIdentifier
+  when defined(rln_v2):
+    ## the external nullifier used for the generation of the `proof` (derived from poseidon([epoch, rln_identifier]))
+    externalNullifier*: ExternalNullifier
```

Comment:
Is is possible to use a different attribute name? `externalNullifier` is also part of the `ProofMetadata` and it is more difficult to follow its references.

Reply:
it's the same field/value though